content/intro-to-storybook/vue/pt/conclusion.md

```diff
@@ -3,31 +3,32 @@ title: 'Conclusão'
 description: 'Fusão de todo o conhecimento adquirido e aprendizagem de técnicas extra de Storybook'
 ---
 
-Parabéns! Acabaste de criar o teu primeiro interface de utilizador com Storybook. Durante esta jornada aprendeste como construir, compor, testar e implementar componentes de interface de utilizador.
-Se estiveste a seguir o tutorial, o repositório e o Storybook implementado irá ser idêntico a este:
+Parabéns! Você criou sua primeira interface do usuário no Storybook. Ao longo do caminho, você aprendeu a criar, compor, testar e implantar componentes da interface do usuário. Se você seguiu o tutorial, seu repositório e o Storybook implantado devem ficar assim:
 
 [📕 **GitHub repo: chromaui/learnstorybook-code**](https://github.com/chromaui/learnstorybook-code)
 <br/>
 [🌎 **Storybook implementado**](https://clever-banach-415c03.netlify.com/)
 
-O Storybook é uma ferramenta bastante poderosa para React, Vue, Angular.
-É composta por uma comunidade de desenvolvimento próspera e um número grande de extras. Esta introdução somente arranha a superfície das potencialidades existentes. Estamos confiantes que depois da adoção do Storybook, irá ficar impressionado com o quanto é produtivo a construção de interfaces de utilizador duradouros.
+O Storybook é uma ferramenta poderosa para React, Vue e Angular.
+Possui uma comunidade de desenvolvedores próspera e uma grande variedade de complementos. Esta introdução arranha a superfície do que é possível. Estamos confiantes de que, ao adotar o Storybook, você ficará impressionado com a produtividade da criação de UIs duráveis.
 
 ## Saber mais
 
-Pretende saber mais? Aqui ficam alguns recursos que irão ajudar.
+Pretende saber mais? Aqui ficam algumas dicas que irão ajudar.
 
-- [**Documentação Oficial Storybook**](https://storybook.js.org/basics/introduction/) contem a documentação da API, links da comunidade e a galeria de extras.
+- [**Documentação Oficial Storybook**](https://storybook.js.org/basics/introduction/) contem a documentação da API, links da comunidade e a galeria de complementos.
 
 - [**Manual de testes visuais**](https://blog.hichroma.com/the-delightful-storybook-workflow-b322b76fd07)
-  enumera as boas práticas de fluxo de trabalho, usadas por equipas que operam a alta velocidade, tal como por exemplo a SquareSpace, a Major League Soccer, a Discovery Network e Apollo GraphQL.
+  enumera as boas práticas de fluxo de trabalho, usadas por equipes que operam a alta velocidade, tal como por exemplo a SquareSpace, a Major League Soccer, a Discovery Network e Apollo GraphQL.
```

Comment:
this is one of those....., i know that the wording in brazilian portuguese is "equipes", if you don't mind leaving it as is for the time being. The choice of words needs to be better researched so that the translation doesn't break it's meaning and can appeal to both readers.